src/sentry/models/integration.py

```diff
@@ -28,14 +28,14 @@ class DocIntegration(Model):
 
     __include_in_export__ = False
 
-    name = models.CharField(max_length=255)
+    name = models.CharField(max_length=64)
     slug = models.CharField(max_length=64, unique=True)
     author = models.CharField(max_length=255)
     description = models.TextField()
     url = models.URLField()
     popularity = models.PositiveSmallIntegerField(null=True, default=1)
     # Quickly hide/show integration, used for logo uploads
-    is_draft = models.NullBooleanField(default=True)
```

Comment:
I know altering a column type can be dangerous, but there are no rows in this table yet.

Reply:
In this case it's fine, the type of the column just remains `bool`